tests/api/test_get_resource.py

```diff
@@ -587,10 +560,8 @@ def test_get_resources(client, resource, uid, expected):
 
     if resource == 'advisory':
         advisory.assigned_to.connect(mprahl)
-        advisory.package_owner.connect(tbrady)
```

Comment:
So sad that you're removing a connection with Tom Brady :cry:

Reply:
Nick Foles is the new GOAT sorry :1st_place_medal:

Reply:
Tom Brady is the :goat:!